Mathlib/CategoryTheory/Adjunction/Mates.lean

```diff
@@ -108,13 +111,58 @@ def transferNatTrans : (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R
       h.naturality, -Functor.map_comp, ← Functor.map_comp_assoc G, R₂.map_comp]
 #align category_theory.transfer_nat_trans CategoryTheory.transferNatTrans
 
+-- ER: A new construction with a new name.
+def Mates :
+    (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R₂) where
+      toFun := by
+        intro α
+        have R₁Gη₂ := whiskerLeft (R₁ ⋙ G) adj₂.unit
+        have R₁αR₂ := whiskerRight (whiskerLeft R₁ α) R₂
+        have ε₁HR₂ := whiskerRight adj₁.counit (H ⋙ R₂)
+        exact R₁Gη₂ ≫ R₁αR₂ ≫ ε₁HR₂
```

Comment:
```suggestion
      toFun := fun α => whiskerLeft (R₁ ⋙ G) adj₂.unit ≫ whiskerRight (whiskerLeft R₁ α) R₂ ≫ whiskerRight adj₁.counit (H ⋙ R₂)
```
writing it this way could save a `dsimp` when it comes to proofs - it's also closer to simp-normal form.
Similar for `invFun`.